Metalbrot.playground/Contents.swift

```diff
@@ -165,6 +161,8 @@ metalView.addSubview(Label(string: "Click me!"), at: CGPoint(x: 5, y: 5))
 
 PlaygroundPage.current.liveView = metalView
 
+controller.viewDidLayout()
```

Comment:
What is this for? It still seems to work without this change on my machine.

Reply:
It is indeed a hack. On my machine `threadgroupSizes` stay zero without it, but that is most probably just a glitch on my side. Good thing is to remove it.